Gemfile.lock

```diff
@@ -57,8 +57,8 @@ GEM
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
     mercenary (0.3.6)
-    mini_portile2 (2.8.8)
-    nokogiri (1.18.8)
+    mini_portile2 (2.8.9)
```

Comment:
Not sure why this extra change is here, though the changelog seems minor: https://github.com/flavorjones/mini_portile/blob/main/CHANGELOG.md#289--2025-05-12